netlify-tutorial.md

```diff
@@ -30,10 +30,12 @@ Follow the instructions below to create your free PostgreSQL and Redis services
 
 1. Select **Free** plan.
 1. Click **Create free service**.
+1. Install the [GitHub CLI](https://github.com/cli/cli#installation).
+1. Install the [Netlify CLI](https://docs.netlify.com/cli/get-started/).
 
 # Step 2. Get the application code
 
-To deploy the application to Netlify you'll need to have your own GitHub repository for [aiven/nextjs-netlify](https://github.com/aiven/nextjs-netlify). If you have the [GitHub CLI](https://github.com/cli/cli#installation) installed, you can simply follow the instructions below to fork the repository. Otherwise, see the relevant [GitHub instructions](https://docs.github.com/en/get-started/quickstart/fork-a-repo?tool=webui#forking-a-repository).
+To deploy the application to Netlify you'll need to have your own GitHub repository for [aiven/nextjs-netlify](https://github.com/aiven/nextjs-netlify). See the relevant [GitHub CLI instructions to fork the repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo?tool=webui#forking-a-repository).
```

Comment:
```suggestion
To deploy the application to Netlify you'll need to have your own GitHub repository for [aiven/nextjs-netlify](https://github.com/aiven/nextjs-netlify). See the relevant [GitHub CLI instructions to fork the repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo?tool=cli#forking-a-repository).
```

The previous link directed to the GitHub UI instructions, but now we refer to CLI instructions, so link needs to be changed.